monitoring/prometheus/alerts/ceph_default_alerts.yml

```diff
@@ -0,0 +1,154 @@
+groups:
+  - name: cluster health
+    rules:
+      - alert: health error
+        expr: ceph_health_status == 2
+        for: 5m
+        labels:
+          severity: critical
+          type: ceph_default
+        annotations:
+          description: Ceph in error for > 5m
+      - alert: unhealthy
+        expr: ceph_health_status != 0
+        for: 15m
+        labels:
+          severity: warning
+          type: ceph_default
+        annotations:
+          description: Ceph not healthy for > 15m
+  - name: mon
+    rules:
+      - alert: low monitor quorum count
+        expr: sum(ceph_mon_quorum_status) < 3
+        labels:
+          severity: critical
+          type: ceph_default
+        annotations:
+          description: Monitor count in quorum is low
+  - name: osd
+    rules:
+      - alert: 10% OSDs down
+        expr: sum(ceph_osd_up) / count(ceph_osd_in) <= 0.9
+        labels:
+          severity: critical
+          type: ceph_default
+        annotations:
+          description: More then 10% of OSDS are down
+      - alert: OSD down
+        expr: count(ceph_osd_up == 0) > 1
```

Comment:
Shouldn't this be `count(ceph_osd_up == 0) > 0`? Alternatively, you can use just `ceph_osd_up < 0.5` or `ceph_osd_up == 0` -- prometheus will create an alert iff the expression returns a series.